components/bl-radio-buttons-list-component/styles/index.less

```diff
@@ -0,0 +1,68 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-radioButtonsList-themeColor: @themePrimary;
+@bl-customComponent-radioButtonsList-backgroundColor: @appBackgroundColor;
+@bl-customComponent-radioButtonsList-textColor: @appTextColor;
+
+@bl-customComponent-radioButtonsList-inputRadio-width: 42px;
+
+@bl-customComponent-radioButtonsList-radioButtonIcon-width: 24px;
```

Comment:
If we have an variable to change radio button size - we should add possibility to change font size too